src/nxt_cert.c

```diff
@@ -1178,10 +1188,23 @@ nxt_cert_store_delete_handler(nxt_task_t *task, nxt_port_recv_msg_t *msg)
 {
     u_char           *p;
     nxt_str_t        name;
+    nxt_port_t       *ctl_port;
     nxt_runtime_t    *rt;
     nxt_file_name_t  *path;
 
     rt = task->thread->runtime;
+    ctl_port = rt->port_by_type[NXT_PROCESS_CONTROLLER];
+
+    if (nxt_slow_path(ctl_port == NULL)) {
+        nxt_alert(task, "controller port not found");
+        return;
+    }
+
+    if (nxt_slow_path(msg->pid != ctl_port->pid)) {
+        nxt_debug(task, "process %PI cannot delete certificates",
```

Comment:
This message is on `debug` level, whereas similar message is `alert` in store certificates check. Please explain or adjust message level. I think, both should be `alert` or `warning`.